deploy/srep-1000-ethtool-exporter/11-ServiceMonitor.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   labels:
-    monitoring.openshift.io/collection-profile: full
```

Comment:
this will result in invalid yaml

Reply:
Thanks.  I've since removed this file replacing all CMO config for OBO/RHOBS.